components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/pages/src/app/components/Apis/Details/NavBar.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react'
```

Comment:
License header is missing. There are multiple places in the PR. Correct them as well.